include/boost/geometry/io/wkt/write.hpp

```diff
@@ -62,22 +63,67 @@ namespace boost { namespace geometry
 namespace detail { namespace wkt
 {
 
+template <typename Output>
+struct output_formatter
+{
+    template <typename T>
+    static void append(Output& out, T const& v)
+    {
+        out << v;
+    }
+};
+
+template <typename CharT, typename Traits, typename Allocator>
+struct output_formatter<std::basic_string<CharT, Traits, Allocator>>
+{
+    template <typename T>
+    static void append(std::basic_string<CharT, Traits, Allocator>& out, T const& val)
+    {
+        std::basic_stringstream<CharT, Traits, Allocator> ss;
+        ss << val;
+        out += ss.str();
+    }
+
+    static void append(std::basic_string<CharT, Traits, Allocator>& out, CharT c)
+    {
+        out += c;
+    }
+
+    static void append(std::basic_string<CharT, Traits, Allocator>& out, const CharT* s)
+    {
+        out += s;
+    }
+
+    template <typename AppendCharT, typename AppendTraits, typename AppendAllocator>
+    static void append(
+        std::basic_string<CharT, Traits, Allocator>& out, 
+        std::basic_string<AppendCharT, AppendTraits, AppendAllocator> const& str
+        )
```

Comment:
I know I had this overload in my example code but is this really needed? Are strings passed into append anywhere?